ClientApp/src/components/Home.js

```diff
@@ -1,11 +1,11 @@
+
+=======
```

Comment:
what is this for line for? It throws an unexpected token error.

Reply:
I think is some mess I made  from github   :(